source/base/Text.ooc

```diff
@@ -32,7 +32,10 @@ Text: cover {
 		this _count = 0
 		this _owner = Owner Unknown
 	}
-	init: func@ ~fromStringLiteral (string: CString, =_count) {
+	init: func@ ~fromStringLiteral (string: CString) {
+		this init(string, strlen(string))
+	}
+	init: func@ ~fromStringLiteralWithCount (string: CString, =_count) {
```

Comment:
This is only a stopgap solution until the day we have a compiler that can handle text literals: 
```
t"Text Literal"
```